sdk/typescript/src/types/sui-bcs.ts

```diff
@@ -419,15 +419,44 @@ const BCS_0_27_SPEC: TypeSchema = {
   },
 };
 
+// for version <= 0.28.0
+const BCS_0_28_SPEC: TypeSchema = {
+  structs: {
+    ...BCS_0_27_SPEC.structs,
+    TransactionDataV1: {
+      kind: 'TransactionKind',
+      sender: BCS.ADDRESS,
+      gasData: 'GasData',
+      expiration: 'TransactionExpiration',
+    },
+  },
+  enums: {
+    ...BCS_SPEC.enums,
+    TransactionData: {
+      V1: 'TransactionDataV1',
+    }
+  },
+  aliases: {
+    ObjectDigest: BCS.BASE64,
+  },
+};
+
```

Comment:
You don't have to do this anymore since we don't need to maintain backward-compatibility